### PR TITLE
Docker fix

### PR DESCRIPTION
**Pull request by SchrodingersGat**
- Closes https://github.com/inventree/InvenTree/issues/7227
- Fixes missing file in docker image

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -85,12 +85,17 @@ jobs:
           docker run --rm inventree-test invoke --list
           docker run --rm inventree-test gunicorn --version
           docker run --rm inventree-test pg_dump --version
+          docker run --rm inventree-test test -f /home/inventere/init.sh
+          docker run --rm inventree-test test -f /home/inventere/tasks.py
+          docker run --rm inventree-test test -f /home/inventere/gunicorn.conf.py
```
Comment (matmair) [inline .github/workflows/docker.yaml:90]: possible typos
```suggestion
          docker run --rm inventree-test test -f /home/inventree/init.sh
          docker run --rm inventree-test test -f /home/inventree/tasks.py
          docker run --rm inventree-test test -f /home/inventree/gunicorn.conf.py
```

File: .github/workflows/docker.yaml [hunk 2]
```diff
@@ -85,12 +85,17 @@ jobs:
...
+          docker run --rm inventree-test test -f /home/inventree/src/backend/requirements.txt
           docker run --rm inventree-test test -f /home/inventree/src/backend/InvenTree/manage.py
       - name: Build Docker Image
         # Build the development docker image (using docker-compose.yml)
         run: docker compose --project-directory . -f contrib/container/dev-docker-compose.yml build --no-cache
       - name: Update Docker Image
         run: |
+          docker compose --project-directory . -f contrib/container/dev-docker-compose.yml run inventree-dev-server invoke install
           docker compose --project-directory . -f contrib/container/dev-docker-compose.yml run inventree-dev-server invoke update
```
Comment (matmair) [inline .github/workflows/docker.yaml:99]: Comment: I think this is redundant as `inv update` already calls `inv install`

File: contrib/container/Dockerfile [hunk 3]
```diff
@@ -128,6 +128,7 @@ COPY --from=prebuild /root/.local /root/.local
 
 # Copy source code
 COPY src/backend/InvenTree ${INVENTREE_BACKEND_DIR}/InvenTree
+COPY src/backend/requirements.txt ${INVENTREE_BACKEND_DIR}/requirements.txt
```
Comment (matmair) [inline contrib/container/Dockerfile:131]: Comment: Ok that is an embarrassing mistake, that is on me. Interesting that the CI did not fail

**Review by matmair — APPROVED** [review]

File: .github/workflows/docker.yaml (hunk 1 above)
Comment (SchrodingersGat) [reply .github/workflows/docker.yaml]: Intentional to check that the checks actually work :)

File: .github/workflows/docker.yaml (hunk 2 above)
Comment (SchrodingersGat) [reply .github/workflows/docker.yaml:99]: I agree, but 'inv update' was previously *not* failing in these tests..

File: contrib/container/Dockerfile (hunk 3 above)
Comment (SchrodingersGat) [reply contrib/container/Dockerfile:131]: Yes, it has been passing CI for quite some time now! I cannot work out how the failure during `invoke install` is ignored.

File: .github/workflows/docker.yaml (hunk 1 above)
Comment (matmair) [reply .github/workflows/docker.yaml]: Ah, that makes sense. I was really confused about how you would typo the project name

File: contrib/container/Dockerfile (hunk 3 above)
Comment (matmair) [reply contrib/container/Dockerfile:131]: That is interesting - is the raise not working?

Comment (SchrodingersGat) [reply contrib/container/Dockerfile:131]: I only just added the raise

**Pull request merged**